.github/workflows/lint-workflows.yml

```
@@ -1,43 +1,43 @@
-  zizmor:
-    name: zizmor
-    runs-on: ubuntu-latest
-    timeout-minutes: 5
-    permissions:
-      security-events: write
-    steps:
-      - uses: actions/checkout@df4cb1c069e1874edd31b4311f1884172cec0e10 # v6
-        with:
-          persist-credentials: false
-      - uses: zizmorcore/zizmor-action@5f14fd08f7cf1cb1609c1e344975f152c7ee938d # v0.5.6
+name: Lint workflows
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)